recipes/qt-advanced-docking-system/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Does qt-advanced-docking-system require C++14? If yes, we need to validate it.

Reply:
You can find it in their CMakeLists.txt: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/blob/ad06241c9d1e61574191cb7f29c650a90a620f48/src/CMakeLists.txt#L70

I can build it when I replace the C++14 requirement with C++11 using a patch, so I guess its unnecessary. Would that be enough to verify it? It so, I will add the patch.